packages/mui-system/src/Box/Box.spec.tsx

```diff
@@ -10,6 +10,16 @@ function Test(props: TestProps) {
   return <span {...other}>{test}</span>;
 }
 
+declare module './Box' {
+  interface CustomSystemProps {
+    customProp?: 'customValue';
+  }
+}
+
+function CustomSystemPropsTest() {
+  <Box customProp="customValue" />;
+}
+
```

Comment:
TypeScript module augmentation tests should be written in `packages\mui-material\test\typescript\moduleAugmentation\*.spec.tsx`

Reply:
I've placed the test inside of `packages/mui-system/test/typescript/moduleAugmentation` since it deals with the system package. Let me know if I should move it to `mui-material` nonetheless

Reply:
It's fine, but can you update the CI configuration to run the test? `mui-system` module augmentation tests do not run in CI. We need to update `config.yml` to:
```diff
--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -266,6 +266,7 @@ jobs:
             yarn workspace @mui/material typescript:module-augmentation
             yarn workspace @mui/base typescript:module-augmentation
             yarn workspace @mui/joy typescript:module-augmentation
+            yarn workspace @mui/system typescript:module-augmentation
```